Simong expedia book package

gmsimon22 opened the pull request:
Here is my procedure for booking a package on Expedia

Ademola-Adepoju reviewed (commented):
Well done, Justin!

Step 1 is acting like a coaching, when I think it might be a Staging move. Perhaps, you want to consider that.

sezito2 reviewed (approved):
Instead of subheadings, I would put your steps into an ordered list to make it easier to read. I like your images to aid the audience.

gmsimon22 committed:
Added Outline

I added an outline of the content, including the main headers

gmsimon22 committed:
added images

added images to book package image folder

gmsimon22 committed:
Added content

added content to the booking package procedure

gmsimon22 committed:
added images to procedure

I added images to the procedure content

gmsimon22 committed:
fixed an error in code

I fixed an error in the code where the final details button image wasn't appearing

gmsimon22 committed:
changed image markdown to html

I changed the image markdown to html with image sizing

WBlan97 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+
+<img src="/simong/images/images-package/Packages-heading-highlight.jpeg" alt="package heading" style="width: 300px">
+
+2) Enter your travel details, then click 'search'
```
I'd have this as two tasks.

alyssajohnsonunofficial replied:
I actually think these are better left as one step! I think if clicking 'search' is left as its own step, it will become too much of a lower level step because it's kind of a common sense thing that everyone knows to do when they see a 'search' button! I'd almost say you don't even need to include the part about clicking 'search'!

WBlan97 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+<img src="/simong/images/images-package/room-overview.jpeg" alt="room overview" style="width: 300px">
+
+*You can customize your stay, adding breakfast options and refund options if offered.*
+
```
Nice visual cues!

WBlan97 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+
+3) fill out the requested personal and financial information details
+
+4) Choose whether or not to protect your trip
```
Maybe here would be a great place for an alerting move, like what could happen if you don't protect your trip or something along those lines.

dmbrasw2 replied:
Seconded, that's a good idea!

alyssajohnsonunofficial replied:
I agree with Wyatt and Deanna here!

WBlan97 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+5) Select 'complete booking' at the end of the form
+
+<img src="/simong/images/images-package/complete-booking-button.jpeg" alt="complete booking button" style="width: 300px">
+
```
Nice work! I like all the visual cues, and steps and headers are easy to follow. I would maybe add more motivation and staging moves, but the coaching is pretty good as in.

alyssajohnsonunofficial replied:
Agreed that the steps and headers are super easy to follow and the procedure has great visual cues!

WBlan97 commented:
Great work here! I think the coaching stages are super well written and easy to follow. I would consider adding some more staging moves and motivation— such as what makes Expedia better than other methods, what is often included in the packages, maybe even a statistic about price difference or savings etc.

dmbrasw2 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+
+## Finding a Package
+
+1) On Expedia's main page, click on 'packages'
```
Good use of an image!

dmbrasw2 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+
+A list of accommodations will appear, ranging in price and ratings. The total costs per traveler (including flights and accomodations) is included in the price estimate.
+
+1) select an accomodation
```
Suggest capitalizing the first letter of all steps

alyssajohnsonunofficial replied:
Agreed! Also, great start with imperative verbs!

dmbrasw2 commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+
+<img src="/simong/images/images-package/complete-booking-button.jpeg" alt="complete booking button" style="width: 300px">
+
+Once you have completed these steps, a booking confirmation email will be sent to your inbox. Enjoy your travels!
```
Good ending line!

alyssajohnsonunofficial replied:
Agreed, this whole procedure has a great tone and I love the language you use. It feels very user-friendly and fun.

alyssajohnsonunofficial commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+## About Expedia Packages
+Selecting a package on Expedia can be a useful way to secure all trip flights and accommodations at once. Expedia offers thousands of customizable packages to choose from.
```
I love your introduction! It's concise, helpful, tells the user what they're getting out of this guide, and interesting to read! Also, your image choices are great!

alyssajohnsonunofficial commented on simong/p-expedia-book-package.md:
```diff
@@ -0,0 +1,65 @@
+Selecting a package on Expedia can be a useful way to secure all trip flights and accommodations at once. Expedia offers thousands of customizable packages to choose from.
+
+## Prerequisites
+You will need to have a location and budget in mind to make your choices.
```
I almost feel like this prerequisite isn't needed here, but then again I'm basing that on the assumption that the user will already have those things in mind, and that may not be true for everyone! It's a very concise and well-written prerequisite either way!

alyssajohnsonunofficial reviewed (commented):
This one looks great so far, Gabby! I left one or two notes of potential changes, but I already really like this one.

gmsimon22 committed:
Changed image, added captions, re-formated, added more the staging

I changed an image, added captions, re-formated, added more the staging, updated wording, and added alerting moves

gmsimon22 committed:
Added user scenario and readme procedure summary

I added the user scenario to this procedure and created a summary for the readme.md

gmsimon22 committed:
added PDF and google doc links to user scenario

I added the PDF and google doc versions of my task analysis to my user-scenario.md

lingeringcode committed:
Feedback from Lindgren
